docs/core/extensions/logging.md

```diff
@@ -3,34 +3,106 @@ title: Logging in C#
+then add a configured ILoggerFactory object to the host's DI container when the host is built. Before the host is built you can adjust the logging configuration
+via <xref:Microsoft.Extensions.Hosting.HostApplicationBuilder.Logging?displayProperty=nameWithType>, <xref:Microsoft.AspNetCore.Builder.WebApplicationBuilder.Logging?displayProperty=nameWithType>, or similar APIs on other
+hosts. Hosts also apply logging configuration from default configuration sources as *appsettings.json* and environment variables.
 
-- Creates a logger, `ILogger<Worker>`, which uses a log *category* of the fully qualified name of the type `Worker`. The log category is a string that is associated with each log.
-- Calls <xref:Microsoft.Extensions.Logging.LoggerExtensions.LogInformation%2A> to log at the `Information` level. The Log *level* indicates the severity of the logged event.
+This example customizes the ILoggerFactory provided by WebApplicationBuilder. It adds [OpenTelemetry](https://github.com/open-telemetry/opentelemetry-dotnet#getting-started) as a logging provider transmitting the logs over [OTLP](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md#enable-log-exporter):
 
-:::code language="csharp" source="snippets/configuration/worker-service/Worker.cs" range="3-18" highlight="12":::
+:::code language="csharp" source="snippets/logging/aspnet-mvc/Program.cs" range="1-9" highlight="4":::
 
-[Levels](#log-level) and [categories](#log-category) are explained in more detail later in this article.
+### Creating an ILoggerFactory with DI
+
+If you are using a DI container without a host, use <xref:Microsoft.Extensions.DependencyInjection.LoggingServiceCollectionExtensions.AddLogging%2A> to configure and add `ILoggerFactory` to the container.
+
+:::code language="csharp" source="snippets/logging/di-without-host/Program.cs" range="1-29" highlight="6":::
+
+The preceding example:
+
+- Created a DI service container containing an ILoggerFactory configured to write to the console
+- Added a singleton `Worker` to the container
+- Created an instance of the `Worker` from the DI container which also automatically created an `ILogger<Worker>` to use as the constructor argument.
+- Invoked Worker.DoSomeWork() which used the `ILogger<Worker>` to log a message to the console.
 
 ## Configure logging
 
-Logging configuration is commonly provided by the `Logging` section of *appsettings*.`{Environment}`*.json* files. The following *appsettings.Development.json* file is generated by the .NET Worker service templates:
+Logging configuration can be set in code or via external sources such as config files and environment variables. Using external configuration is beneficial when possible because
+it can be changed without rebuilding the application. However some tasks, such as setting logging providers, can only be configured from code.
```

Comment:
Active voice, grammar, and remove extra lines.

```suggestion
Logging configuration is set in code or via external sources, such as, config files and environment variables. Using external configuration is beneficial when possible because it can be changed without rebuilding the application. However, some tasks, such as setting logging providers, can only be configured from code.
```